plugins/source/mysql/client/client.go

```diff
@@ -0,0 +1,70 @@
+package client
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/cloudquery/plugin-sdk/plugins/source"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/go-sql-driver/mysql"
+	"github.com/rs/zerolog"
+)
+
+type Client struct {
+	logger      zerolog.Logger
+	metrics     *source.Metrics
+	Tables      schema.Tables
+	db          *sql.DB
+	tableSchema string
+}
+
+var _ schema.ClientMeta = (*Client)(nil)
+
+func (*Client) ID() string {
+	return "source-mysql"
```

Comment:
Don't we include `c.spec.Name` in case there are multiple sources going into the same destination?

Reply:
I think the `ID` is only used to distinguish clients when multiplexing (which we don't have here):
https://github.com/cloudquery/plugin-sdk/blob/0f58331ea77ea07562a98edd50bd51b156685db9/plugins/source/metrics.go#L60

Reply:
See also https://github.com/cloudquery/cloudquery/blob/34cbb8d99e0f5183bd885023dc8a138322b860ba/plugins/source/postgresql/client/client.go#L41